apps/rush-lib/config/jest.config.json

```diff
@@ -1,3 +1,4 @@
 {
-  "preset": "./node_modules/@rushstack/heft/includes/jest-shared.config.json"
+  "preset": "./node_modules/@rushstack/heft/includes/jest-shared.config.json",
+  "silent": false
```

Comment:
I remember `silent` being somehow incompatible with Heft's logger. Heft is supposed to rewire `console.log` to get printed specially, so that test output cannot get confused with Heft output.

Reply:
https://github.com/microsoft/rushstack/blob/ea805f86b2a85ca073aa6d77a4d7efbddb7087b2/apps/heft/src/plugins/JestPlugin/HeftJestReporter.ts#L92-L98

Reply:
Oops. Didn't mean to commit that.